### PR TITLE
Add has key querier

### PR DESCRIPTION
**Pull request by njmurarka**
CM-99

Note that I created this branch off the "cleanup_nsd" branch. Not sure if this is a good idea or not.

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -200,7 +200,7 @@ func NewCRUDApp(
 			app.slashingKeeper.Hooks()),
 	)
 
-	// The NameserviceKeeper is the Keeper from the module for this tutorial
+	// The CrudKeeper is the Keeper from the module
```
Comment (rnistuk) [inline app.go:203]: These changes have already been approved in a previous pull request.

**Review by rnistuk — COMMENTED** [review]
Please squash commits next time, so we only get one commit to review.

File: x/crud/alias.go [hunk 2]
```diff
@@ -39,4 +39,5 @@ type (
 	MsgBLZUpdate    = types.MsgBLZUpdate
 	MsgBLZDelete    = types.MsgBLZDelete
 	QueryResultRead = types.QueryResultRead
+	QueryResultHas  = types.QueryResultHas
```
Comment (rnistuk) [inline x/crud/alias.go:42]: Cool, we need a place to hold the result.

**Comment by njmurarka** [comment]
How do I do that? You mean ensure a PR is always exactly one commit?

File: x/crud/client/cli/query.go [hunk 3]
```diff
@@ -58,3 +59,21 @@ func GetCmdBLZQRead(queryRoute string, cdc *codec.Codec) *cobra.Command {
 		},
 	}
 }
+
+func GetCmdBLZQHas(queryRoute string, cdc *codec.Codec) *cobra.Command {
```
Comment (rnistuk) [inline x/crud/client/cli/query.go:63]: This looks good.

File: x/crud/internal/keeper/querier.go [hunk 4]
```diff
@@ -49,3 +52,20 @@ func queryRead(ctx sdk.Context, path []string, _ abci.RequestQuery, keeper Keepe
 
 	return res, nil
 }
+
```
Comment (rnistuk) [inline x/crud/internal/keeper/querier.go:55]: Cool, returns the binary string equiv of true or false...

File: x/crud/internal/types/querier.go [hunk 5]
```diff
@@ -21,3 +21,11 @@ type QueryResultRead struct {
...
 	return r.Value
 }
+
+type QueryResultHas struct {
+	Value string
+}
+
+func (r QueryResultHas) String() string {
+	return r.Value
+}
```
Comment (rnistuk) [inline x/crud/internal/types/querier.go:39]: The editor is complaining that there is no empty carriage return at the end of the file.

File: x/crud/internal/types/querier.go [hunk 6]
```diff
@@ -21,3 +21,11 @@ type QueryResultRead struct {
 func (r QueryResultRead) String() string {
 	return r.Value
 }
+
+type QueryResultHas struct {
```
Comment (rnistuk) [inline x/crud/internal/types/querier.go:27]: Cool, we need a place to hold the query result. And a nice member function to get the value.

**Review by rnistuk — APPROVED** [review]
Please squash commits next time, so we only get one commit to review. Also reference a Jira issue in the commit message.

**Pull request merged**